components/layout/Container.tsx

```diff
@@ -0,0 +1,42 @@
+import { styled } from '@stitches/react';
```

Comment:
breakpoint가 적용되지 않는 이유는 라이브러리를 직접 불러와서 그렇습니다. 🥲
저희가 설정한 stitches theme에서 나오는 `styled` 함수를 써야 테마가 제대로 적용됩니당.

```suggestion
import { styled } from 'stitches.config';
```

Reply:
헐!! 천재....!!!!! 감사합니닼ㅋㅋㅋㅋ 🥹